sql/information_schema/information_schema.go

```diff
@@ -1907,27 +1911,27 @@ func tablesRowIter(ctx *Context, cat Catalog) (RowIter, error) {
 
 		for _, view := range views {
 			rows = append(rows, Row{
-				"def",     // table_catalog
-				db.Name(), // table_schema
-				view.Name, // table_name
-				"VIEW",    // table_type
-				nil,       // engine
-				nil,       // version (protocol, always 10)
-				nil,       // row_format
-				nil,       // table_rows
-				nil,       // avg_row_length
-				nil,       // data_length
-				nil,       // max_data_length
-				nil,       // max_data_length
-				nil,       // data_free
-				nil,       // auto_increment
-				y2k,       // create_time
-				nil,       // update_time
-				nil,       // check_time
-				nil,       // table_collation
-				nil,       // checksum
-				nil,       // create_options
-				"VIEW",    // table_comment
+				"def",      // table_catalog
```

Comment:
This isn't quite right: the catalog is the DB name when the schema is provided. E.g. with the default postgres setup, the catalog and schema names will be `postgres`, `public`